lib/vagrant-service-manager/command.rb

```diff
@@ -5,6 +5,7 @@ module Vagrant
   module ServiceManager
     DOCKER_PATH = '/home/vagrant/.docker'
     SUPPORTED_SERVICES = ['docker', 'openshift', 'kubernetes']
+    SCCLI_SERVICES = ["openshift", "kubernetes", "k8s"]
```

Comment:
> Looks like it is done only to make code logic easy?

Although `sccli` understands only `openshift` and `k8s`, the plugin provides an extra option for users to either enter `k8s` or `kubernetes` as a choice of service name to restart.